content/webapp/services/prismic/fetch/stories-landing.ts

```diff
@@ -9,30 +9,46 @@ const graphQuery = `{
     stories {
       story {
         ...on articles {
```

Comment:
nit: I like how with all of your additions you've been consistent with a space after the `...`, but this one (not changed in this PR) doesn't have a space. I spotted this inconsistency in the Prismic docs even. But I reckon we should pick one consistently.

Reply:
love it